src/test/java/io/camunda/testing/util/Utilities.java

```diff
@@ -64,4 +73,28 @@ public static ProcessInstanceEvent startProcessInstance(
     Thread.sleep(100);
     return instanceEvent;
   }
+
+  public static PublishMessageResponse sendMessage(
+      final ZeebeClient client, final String messsageName, final String correlationKey)
+      throws InterruptedException {
+    return sendMessage(client, messsageName, correlationKey, Duration.ofDays(99999));
```

Comment:
:thought_balloon: I think we use -1 or some such value to indicate "forever"

Reply:
I've tested this and this does not seem to be the case. When I enter -1 days the messages expire instantly.